### PR TITLE
Enabling syntax highlighting

### PR DESCRIPTION
**Pull request by secondsun**
Enables syntax highlighting, updates kramdown, and includes a "sample" blog post demonstrating it.

### CODE REVIEW

**Comment by secondsun** [comment]
How to test:
You will have to run the bundle install scripts to update and install kramdown 1.6 and rouge 1.8.  After that fire up jekyll and look at the Android 2.1 post in the news sections.  You should see something like this : picture

**Comment by secondsun** [comment]
Also click around and make sure I didn't break anything.  One of the options enabled github style markdown which may break other things.

File: _posts/2015-03-25-aerogear-android-2.1.0.md [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+  android:name="serverPath"
+  android:value="/sync" />
+</service>
+{% endhighlight %}
+
+Any Activty you want to receive sync events needs to implement the `SyncServerConnectionListener` interface, bind itself to the `SyncService` instance, and then subscribe to events on connections.
+
+Binding and subscribing example : 
+
+{% highlight java %}
```
Comment (edewit) [inline _posts/2015-03-25-aerogear-android-2.1.0.md]: This is not git style markdown, but that is now also supported?

Comment (secondsun) [inline _posts/2015-03-25-aerogear-android-2.1.0.md]: Yes

**Comment by secondsun** [comment]
@matzew @lfryc care to :eyes: ?

**Comment by matzew** [comment]
:+1: I like the new syntax highlighting!

**Comment by danielpassos** [comment]
:+1:

**Comment by secondsun** [comment]
landed

**Pull request merged**